### PR TITLE
Lkim/job status

### PR DESCRIPTION
**Pull request by laurenkim1**

### CODE REVIEW

File: request-manager/request/manager.go [hunk 1]
```diff
@@ -542,6 +542,24 @@ func (m *manager) getWithJc(requestId string) (proto.Request, error) {
 	}
 
 	req.JobChain = &jc
+
+	var params proto.CreateRequestParams
+	var rawParams []byte // raw job chains are stored as blobs in the db.
+	q = "SELECT request FROM raw_requests WHERE request_id = ?"
```
Comment (michaelfinch) [inline request-manager/request/manager.go]: How could we get this info without making an extra call to the database?

Comment (laurenkim1) [reply request-manager/request/manager.go]: I thought the next line was a call to the database?

File: spinc/cmd/status.go [hunk 2]
```diff
@@ -40,7 +40,21 @@ func (c *Status) Run() error {
 		return nil
 	}
 
+	running := []string{}
```
Comment (michaelfinch) [inline spinc/cmd/status.go]: make this a map of [jobName] => status instead?

File: spinc/cmd/status.go [hunk 3]
```diff
@@ -40,7 +40,21 @@ func (c *Status) Run() error {
...
+	statuses := []string{}
```
Comment (michaelfinch) [inline spinc/cmd/status.go]: instead of using two slices, you could make a map of [jobName] => status

**Pull request merged**